src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -360,6 +367,9 @@ def with_subqueries(self, subquery_plans: List["SnowflakePlan"]) -> "SnowflakePl
 
     @cached_property
     def attributes(self) -> List[Attribute]:
+        assert (
```

Comment:
@sfc-gh-aalam though highly unlikely, is it possible that schema_query can be None or `""` before? Just for the risk here, maybe just leave it as a comment instead of assertion?

Reply:
@sfc-gh-jdu I don't think that is possible, if you look at the implementation for analyze_attributes https://github.com/snowflakedb/snowpark-python/blob/main/src/snowflake/snowpark/_internal/analyzer/schema_utils.py#L71
when schema_query is None, it will raise an error when it does sql.strip().lower(). if it is "", it will error out when we call session._get_result_attributes(sql). 
With this change, we make the None case more obvious.